libs/blocks/marquee/marquee.css

```diff
@@ -1,9 +1,45 @@
+.marquee.large .foreground.container .text {
+  margin: var(--spacing-m) auto;
+  width: var(--grid-container-width);
+}
+
+.marquee.large .container {
+  width: 100%;
+  padding: 0;
+  gap:0;
+}
+
+.marquee .icon-area.icon-area-multiple > :not(:last-child) img {
+  margin: 0 var(--spacing-s) 0 0;
+}
+
+.marquee.large .text .heading-xxl {
+  margin-bottom: var(--spacing-s);
+}
+
+.marquee.large .text .body-xl:not(.supplemental-text){
+  margin-bottom: 0;
```

Comment:
supplemental-text is already overriding margin bottom to 0 below in this css. We do not neet additional :not operator

Reply:
Fixed this, thanks!